packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -25,6 +25,11 @@ Parameters:
       S3 key, or the ID/ARN of an existing key.
     Default: ''
     Type: String
+  ImageAssetsEcrKmsKeyId:
+    Description: Empty to create a new key, 'AWS_DEFAULT_KEY' to use default
```

Comment:
This description is not in line with the if-else cascade above, empty would "default" to:
```
(params.ecrKey === undefined) {
      // use an AWS managed key
      kmsEcrKeyId = USE_DEFAULT_KEY;
    }
```
right?